README.md

```diff
@@ -20,4 +20,8 @@ See the [codemod CLI doc](https://go.codemod.com/cli-docs) for a full list of av
 
 ## Available codemods
 
-All Node.js codemods are also available in the [Codemod Registry](https://codemod.link/nodejs-official).
+You can find official Node.js codemods in the [Codemod Registry](https://codemod.com/registry?author=nodejs). Additionally, community-contributed Node.js codemods are available in the [Codemod Registry](https://codemod.link/nodejs-official).
```

Comment:
This is not correct: the author is supposed to be the actual author. This is a bug in the codemod registry @alexbit-codemod